addOns/scripts/src/main/java/org/zaproxy/zap/extension/scripts/ScriptConsoleOptionsPanel.java

```diff
@@ -118,13 +134,37 @@ private JComboBox<DefaultScriptChangedBehaviour> getDefaultScriptChangedBehaviou
         return defaultScriptChangedBehaviour;
     }
 
+    private JPanel getFontPanel() {
+        if (fontPanel == null) {
+            fontPanel = new JPanel(new GridBagLayout());
+            fontPanel.setBorder(
+                    new TitledBorder(Constant.messages.getString("scripts.options.font.title")));
+            int row = -1;
+            var fontNameLabel =
+                    new JLabel(Constant.messages.getString("scripts.options.font.fontName"));
+            fontPanel.add(
+                    fontNameLabel, LayoutHelper.getGBC(0, ++row, 1, 0.5, new Insets(2, 4, 2, 2)));
```

Comment:
If you have to increment before using it why not just initialize at zero to start with?

Reply:
This is more uniform, since the first element added to each row contains `++row`. 
If we ever want to add new rows above it, we wouldn't have to change this line.